lib/lbt/analyzer/analyzeLibraryJS.js

```diff
@@ -19,13 +27,17 @@ async function analyze(resource) {
 		interfaces: []
 	};
 
+	const code = await resource.getBuffer();
+	const rootNode = parseJS(code);
+	let sapUiDefineCall = null;
+	visit( rootNode );
+
 	function visit(node) {
-		if ( node.type == Syntax.CallExpression &&
-				node.callee.type === Syntax.MemberExpression &&
-				isMethodCall(node.callee.object, CALL__SAP_UI_GETCORE) &&
-				isIdentifier(node.callee.property, "initLibrary") &&
-				node.arguments.length === 1 &&
-				node.arguments[0].type === Syntax.ObjectExpression ) {
+		// Define call would always be at the top of the tree, so it'd
+		// be the first significant thing that's been found.
+		sapUiDefineCall = sapUiDefineCall || analyzeSapUiDefineCalls(node);
+
+		if (isInitLibraryCall(node)) {
 			node.arguments[0].properties.forEach( (prop) => {
```

Comment:
I can't put a comment below, but the `log.error` could be improved to properly reflect the actual call or maybe mention something more general than `sap.ui.getCore().initLibrary`.

Reply:
Thanks!